modules/nw-sriov-dpdk-example-mellanox.adoc

```diff
@@ -59,84 +60,84 @@ After the configuration update is applied, all the pods in the `openshift-sriov-
 $ oc create -f mlx-dpdk-node-policy.yaml
 ----
 
-. Create the following `SriovNetwork` object, and then save the YAML in the `mlx-dpdk-network.yaml` file.
+. Save the following `SriovNetwork` YAML configuration to an `mlx-dpdk-network.yaml` file:
 +
 [source,yaml]
 ----
 apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetwork
 metadata:
-  name: mlx-dpdk-network
-  namespace: openshift-sriov-network-operator
+name: mlx-dpdk-network
```

Comment:
This YAML example still needs relative indentation, but the left-most level should have no spaces before it.